src/components/OfflineWithFeedback.tsx

```diff
@@ -35,6 +35,9 @@ type OfflineWithFeedbackProps = ChildrenProps & {
     /** Whether we should show the error messages */
     shouldShowErrorMessages?: boolean;
 
+    /** */
+    shouldShowErrorOnTop?: boolean;
+
```

Comment:
What is this for? Are we introducing a new design pattern here?

Reply:
Thanks for the link. That looks kinda odd to me, I'd think we show the error in the push input, not the selection list. @Expensify/design is this the pattern we're using for displaying errors with push inputs?

Reply:
Hmm can we see the screen that shows immediately before this just to confirm? Maybe we don't have space in the screen before for the error message or something?

@trjExpensify might have more context here too.

Reply:
Yeah, we can absolutely place the error message below the push row here. It's what we do on expenses for instance: 
![CleanShot 2024-03-08 at 08 41 00@2x](https://github.com/Expensify/App/assets/2319350/0399d815-316a-4a1c-bb6a-db80c46d8df6)

So let's do that and remove the error message from the select list screen?

Reply:
cc @shawnborton Would you mind taking a look at the [above](https://github.com/Expensify/App/pull/37629#discussion_r1517919126) screenshots to check the design on the error message here? I'm wondering if it's correct to display the error full width as it doesn't match the row immediately above.

Reply:
cc @Expensify/design 

But yeah, I can see where it makes sense to make the width of this error message match the row width above it. So I would be down with that change to make it more clear personally.